WordPressKitTests/WordPressOrgXMLRPCApiTests.swift

```diff
@@ -41,4 +42,224 @@ class WordPressOrgXMLRPCApiTests: XCTestCase {
         self.waitForExpectations(timeout: 2, handler: nil)
     }
 
+    func test404() {
+        stub(condition: isXmlRpcAPIRequest()) { _ in
+            HTTPStubsResponse(data: Data(), statusCode: 404, headers: ["Content-Type": "application/xml"])
```

Comment:
Nitpick: `"Content-Type": "application/xml"]` is used many times here. 

What do you think of DRYing it the next time the tests are touched?

```
let xmlContentTypeHeaders = ["Content-Type": "application/xml"]
```

Reply:
Addressed in https://github.com/wordpress-mobile/WordPressKit-iOS/pull/705/commits/107ddf84296ab25e2b5648f77fd6b54e35656f25